README.md

```diff
@@ -146,6 +146,7 @@ This provider exposes quite a few provider-specific configuration options:
   for credentials.
 * `block_device_mapping` - Amazon EC2 Block Device Mapping Property
 * `elb` - The ELB name to attach to the instance.
+* `unregister_ELB_from_Az` - Removes the ELB from the AZ on removal of the last instance if true (default). In non default VPC this has to be false.
```

Comment:
Will you change this to all lower-case? the awkward casing doesn't match up with the rest of our settings